torch/csrc/jit/script/compiler.cpp

```diff
@@ -2583,6 +2584,11 @@ struct to_ir {
         ->output();
   }
 
+  Value* emitStringIndex(const SourceRange& loc, Value* string, Value* index) {
+    auto string_type = string->type()->cast<StringType>();
```

Comment:
if you have a schematized insert, this function is not necessary. Also, I think this line is unused